cli/src/format.rs

```diff
@@ -95,6 +95,65 @@ impl Format for PolicyEvaluationResponse {
     }
 }
 
+impl Format for PolicyEvaluationResponseRaw {
+    fn pretty<W: Write>(&self, writer: &mut W) {
+        let _ = writeln!(writer);
+
+        // Print summary status.
+        let status = if self.is_failure {
+            style("FAILURE").red()
+        } else if self.incomplete_packages_count > 0 {
+            style("INCOMPLETE").yellow()
+        } else {
+            style("SUCCESS").green()
+        };
+        let _ = writeln!(writer, "Phylum Supply Chain Risk Analysis — {status}\n");
+
+        // Print number of unprocessed packages.
+        if self.incomplete_packages_count > 0 {
+            let pluralization = if self.incomplete_packages_count == 1 { "" } else { "s" };
+            let unprocessed_text =
+                format!("{} unprocessed package{}", self.incomplete_packages_count, pluralization);
+            let incomplete_message = format!(
+                "The analysis contains {}, preventing a complete risk analysis. Phylum is \
+                 currently processing these packages and should complete soon. Please wait for up \
+                 to 30 minutes, then re-run the analysis.\n",
+                style(unprocessed_text).yellow(),
+            );
+            let _ = writeln!(writer, "{}", textwrap::fill(&incomplete_message, 80));
+        }
+
+        // Write summary for each issue.
+        for package in &self.dependencies {
+            for rejection in package.rejections.iter().filter(|rejection| !rejection.suppressed) {
+                let severity = match rejection.source.domain.as_str() {
+                    "author" => "AUT",
+                    "engineering" => "ENG",
+                    "malicious_code" | "malicious" => "MAL",
+                    "vulnerability" => "VUL",
+                    "license" => "LIC",
+                    _ => "UNK",
+                };
+                let message = format!("[{severity}] {}", rejection.title);
```

Comment:
Hmm... You're counting on `rejection.title` to contain the name of the package...

That's definitely not guaranteed for policy-based rejections. But it seems to be true for every issue-based rejection I can find...

We probably need to discuss this with @louislang before relying on it.

Reply:
I just don't want everything duplicated. The UI currently does the same. This PR's format is inspired by what the UI shows at the moment. I'm happy to change things in the future.

Reply:
I believe that will change when the UI is updated to use this endpoint. And maybe the titles will no longer contain the package names?

Reply:
I believe that will change when the UI is updated to use this endpoint. And maybe the titles will no longer contain the package names?